crates/swc_ecma_transforms_module/tests/common_js.rs

```diff
@@ -2470,7 +2470,7 @@ let diffLevel = 0;
 exports.diffLevel = diffLevel;
 
 function diff() {
-  if (!(exports.diffLevel = diffLevel = +diffLevel - 1)) {
+  if (!(--diffLevel, exports.diffLevel = diffLevel)) {
```

Comment:
Is this the correct change?
I checked babel but babel does not do this

Reply:
Babel's behavior is not right, I will open an issue later.
The current behavior of this PR is equivalent to the (fixed) babel behavior with `assumptions.pureGetters=true`.

Reply:
This case looks like it could be changed to `exports.diffLevel = --diffLevel`.
I'll give it a try.